test/scripts/start-services.sh

```diff
@@ -41,11 +41,31 @@ start_parachain()
     pushd ../parachain
 
     cargo build
-    target/debug/artemis-node --tmp --dev >$logfile 2>&1 &
+
+    echo "Generating Parachain spec"
+    target/debug/artemis-node build-spec --dev --disable-default-bootnode > $configdir/spec.json
+
+    echo "Inserting Ganache chain info into genesis spec"
+    ethereum_initial_header=`curl http://localhost:8545 \
```

Comment:
Rather use `$( ... )` instead of backticks for command substitution. The latter has better behavior and is POSIX-compatible: https://stackoverflow.com/questions/9449778/what-is-the-benefit-of-using-instead-of-backticks-in-shell-scripts